helm/velero/values.yaml

```diff
@@ -75,7 +84,7 @@ installCRDs: true
 ##
 configuration:
   # Cloud provider being used (e.g. aws, azure, gcp).
-  provider:
+  provider: aws
```

Comment:
otherwise the CI fail because deployment is not there. Can I use default values only for abs? @giantswarm/team-halo